### DISCUSSION TITLE
Deno 2 and importing the Local API

### DISCUSSION THREAD

**Discussion by torse** [payloadcms/payload#8640] [Q&A]
I am stoked that with yesterdays release of [Deno v2](https://deno.com/blog/v2.0) it is now possible to create and run Payload natively in Deno!
```
$ deno run -A npm:create-payload-app@beta
$ deno run dev
Task dev cross-env NODE_OPTIONS=--no-deprecation next dev
  ▲ Next.js 15.0.0-canary.173
  - Local:        http://localhost:3000
  - Environments: .env
  - Experiments (use with caution):
    · turbo

 ✓ Starting...
 ✓ Ready in 1775ms
 ○ Compiling /admin/[[...segments]] ...
 ✓ Compiled /admin/[[...segments]] in 9.7s (3745 modules)
```
I just ran into a problem running my standalone DB seed script that fails due to an invalid module import on `import config from '@payload-config'`. I am using the code from the docs on [using payload outside Next.js](https://payloadcms.com/docs/beta/local-api/outside-nextjs). I know that Deno is not officially supported by Next/Payload and this is probably a problem in Deno. But I was wondering if anyone knows if there is another way to import payload in plain typescript that also works in Deno? That would be awesome!

`seed.ts`:
```typescript
// We are importing `getPayload` because we don't need HMR
// for a standalone script. For usage of Payload inside Next.js,
// you should always use `import { getPayloadHMR } from '@payloadcms/next/utilities'` instead.
import { BasePayload, getPayload } from 'payload'
import config from '@payload-config'
const payload: BasePayload = await getPayload({ config }) // Get a local copy of Payload by passing your config

process.exit(0)
```

`deno run:`
```bash
$ deno run src/seed/seed.ts
error: [ERR_INVALID_MODULE_SPECIFIER] Invalid module '@payload-config' is not a valid package name imported from '.../my-app/src/seed/seed.ts'
    at file://.../my-app/src/seed/seed.ts:5:20
```

**Comment by runoncedev** [comment]
I'm having a similar issue. Also trying to get a seed script working.

If I run it with Node > v22.9.0 I get the `ERR_INVALID_MODULE_SPECIFIER` error.

If I run with Node =< v22.9.0 I get a `TypeError: Cannot read properties of undefined (reading 'Symbol(pino.wildcardFirst)')` error.
